src/docs/cookbook/animation/animated-container.md

```diff
@@ -12,40 +12,23 @@ js:
     url: https://dartpad.dev/inject_embed.dart.js
 ---
 
-The [`Container`][] class provides a convenient way
-to create a widget with specific properties:
-width, height, background color, padding, borders, and more.
+Lớp [`Container`][] cung cấp một cách thuận tiện để tạo một widget với những thuộc tính riêng biệt sau: width, height, background color, padding, borders, ...
 
-Simple animations often involve changing these properties over time.
-For example,
-you might want to animate the background color from grey to green to
-indicate that an item has been selected by the user.
+Việc đơn giản hoá những hoạt ảnh thường bao gồm thay đổi những thuộc tính này theo thời gian. Ví dụ như bạn muốn chuyển đổi màu của phong nền từ xám đến xanh lá để cho thấy là một mục đã được chọn bởi người dùng.
 
-To animate these properties,
-Flutter provides the [`AnimatedContainer`][] widget.
-Like the `Container` widget, `AnimatedContainer` allows you to define
-the width, height, background colors, and more. However, when the
-`AnimatedContainer` is rebuilt with new properties, it automatically
-animates between the old and new values. In Flutter, these types of
-animations are known as "implicit animations."
```

Comment:
check with the team about "animation", "explicit animation" and "implicit animation"